exercises/bracket-push/src/lib.rs

```diff
@@ -1,13 +1,3 @@
-pub struct Brackets;
-
-impl<'a> From<&'a str> for Brackets {
-    fn from(input: &str) -> Self {
-        unimplemented!("From the '{}' input construct a new Brackets struct", input);
-    }
-}
-
-impl Brackets {
-    pub fn are_balanced(&self) -> bool {
-        unimplemented!("Check if your Brackets struct contains balanced brackets");
-    }
+pub fn brackets_are_balanced(_string: &str) -> bool {
```

Comment:
I understand that the leading underscore is to say it is OK that it is unused.

I would like to suggest instead the model established in https://github.com/exercism/rust/pull/438 - call it `string` and reference it in the `unimplemented` message. The benefit of the approach is that it avoids a potential student misunderstanding, thinking that the variable should remain named `_string`.

Otherwise, this will fall under https://github.com/exercism/rust/issues/476 which asked us to remove all instances of variables with leading underscores

Reply:
Yeah, that sounds reasonable. It felt wrong to put it there in the first place to get past the (denied) warning.